web-dapp/server-lib/logger.js

```diff
@@ -1,10 +1,30 @@
+'use strict';
+const winston = require('winston');
+
+const transports = [];
+
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
why exclude `test` environment here?

Reply:
This is so the logs are not displayed via console on tests. We could define another transport for tests for example.

Example: